dist/api_data.py

```diff
@@ -272,6 +272,10 @@ def __ge__(self, other):
         the file is read-only. All methods that may modify a file are
         disabled. See @ref readonly for more information''',
         type='boolean'),
+    Config('tiered_object', 'false', r'''
+        this file is a tiered object. When opened on its own, it is marked as
+        readonly and may be restricted in other ways''',
+        type='boolean', undoc=True),
```

Comment:
Yes - the extra functionality currently is no checkpoint.  It's also likely that we shouldn't allow opening of this object on its own even as readonly (after all, it may have references to other objects, and the Block manager won't know what to do without viewpoint offered when opening "tiered:").  I'll remove readonly.

Reply:
As it turns out, we'll need readonly independently, as `"object:"` uris "inherit" from `"file:"` and objects need readonly.  So I'm going to leave this as is.

Reply:
Thanks for researching it.